tests/torch2/utils.py

```diff
@@ -37,3 +41,22 @@ def compare_with_reference_file(text_data: str, ref_path: Path, regen_ref_data:
         f"Data mismatch between actual data and reference file: {ref_path}\n"
         f"Actual data and reference data differ. Please review the file contents."
     )
+
+
+def get_reference_graph(graph: NNCFGraph) -> nx.DiGraph:
```

Comment:
Is it possible to reuse the `nncf_graph.get_graph_for_structure_analysis` method here?

Reply:
No, we have already talked about this.